src/Analyzers/CSharp/Tests/UseSimpleUsingStatement/UseSimpleUsingStatementTests.cs

```diff
@@ -1639,6 +1639,52 @@ void M()
         // intentionally empty
     }
 }",
+parseOptions: CSharp8ParseOptions);
+        }
+
+        [WorkItem(58911, "https://github.com/dotnet/roslyn/issues/58911")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseSimpleUsingStatement)]
+        public async Task TestUsingWithoutSpace()
+        {
+            await TestInRegularAndScriptAsync(
+@"public class Test
+{
+    public IEnumerable<Test> Collection { get; } = new[]
+    {
+        new Test()
+        {
+            Prop = () =>
+            {
+                [||]using(var x = Get())
+                {
+                    int i = 0;
+                }
+            }
+        }
+    };
+
+    public Action? Prop { get; set; }
+    public static IDisposable Get() => throw new NotImplementedException();
+}
+",
+@"public class Test
+{
+    public IEnumerable<Test> Collection { get; } = new[]
+    {
+        new Test()
+        {
+            Prop = () =>
+            {
+                using var x = Get();
+                    int i = 0;
```

Comment:
💭 Seems strange that the indentation of this didn't change, but that wouldn't be caused by anything in this PR.

Reply:
@sharwell I filed https://github.com/dotnet/roslyn/issues/62123 for this.